elastic/kibana
# [Fleet] Fix empty assets on package install

## Summary

Resolve #125362 

Show assets after a fresh install from the integration details settings screen.

## How to test?

1. Navigate to an integration
1. Go to the settings tab
1. Install the integration
1. Go to the assets tab
1. Notice assets
1. refresh and the assets are still here.
